src/pretix/plugins/checkinlists/exporters.py

```diff
@@ -159,7 +159,7 @@ def _get_queryset(self, cl, form_data):
             ), self.event.timezone)
             qs = qs.filter(subevent__date_from__lt=dt)
 
-        o = tuple()
+        o = ()
```

Comment:
Why do you somethings think adding tuple is good (see above) and sometimes that removing it is good?

Reply:
I don't get you, I have removed all unnecessary `tuple()` calls.

Reply:
Sorry, misread the diff. Disregard